### ISSUE TITLE
Localization files not used

### ISSUE THREAD

**Issue by andreaslindahl** [mattt/FormatterKit#201]
I have added Swedish as a language to my app, but FormatterKit doesn't seem to use the translations in the FormatterKit.strings files - the default is always used. Do I need to configure anything to make it work?

I'm using Cocoapods, by the way.

**Comment by mliberatore** [comment]
@andreaslindahl, I'm experiencing the same issue on my end. I've noticed that replacing `NSLocalizedStringFromTable` in `FormatterKit`, which uses `[NSBundle mainBundle]`, with `NSLocalizedStringFromTableInBundle`, passing in `[NSBundle bundleForClass:[self class]]` seems to work, leading me to believe this is an issue with using `FormatterKit` via Cocoapods as a framework (see [here](http://blog.cocoapods.org/Pod-Authors-Guide-to-CocoaPods-Frameworks/), regarding `NSBundle`).

**Comment by yas375** [comment]
I believe this has been fixed in this commit: c3e95a368ed186766da4f5cace8f55ef70a5e17e

```
pod 'FormatterKit', :git => 'https://github.com/mattt/FormatterKit.git', :commit => 'c3e95a3'
```
